_assets/js/utils/gaUtil.js

```diff
@@ -19,6 +19,8 @@ function gtag() {
     ['.learn-link', 'click', sendGAClickEvent],
     ['.crt-sidenav-subnav-item', 'click', sendGAClickEvent],
     ['.info-box-link', 'click', sendGAClickEvent],
+    ['.best-bet', 'click', sendGAClickEvent],
```

Comment:
any way to include info on which best bet they've clicked?

Reply:
Yes! The event will include the inner text of the link and the link address